db/schema.rb

```diff
@@ -507,8 +511,8 @@
     t.integer  "source_id"
     t.string   "source_type"
     t.text     "details"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
```

Comment:
:warning: this is not a formatting change like the ones above. Could you take it out? although it's weird that your schema ended up like this :thinking:

Reply:
oops! I rebased the PR and remove this :+1: